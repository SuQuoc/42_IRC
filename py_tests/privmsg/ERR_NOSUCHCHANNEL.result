:AfterLife 001 client0 :Welcome to the Internet Relay Network, user0
