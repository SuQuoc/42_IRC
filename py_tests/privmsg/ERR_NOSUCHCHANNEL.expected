:AfterLife 001 client0 :Welcome to the Internet Relay Network, user0
:AfterLife 412 client0 :No text to send
:AfterLife 403 client0 #nonexistentchannel :No such channel
